src/js/conduit/__tests__/Router/static.methods.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+
+const mockConsole = require('jest-mock-console');
+
+module.exports = function ({
+  utils,
+  Paho,
+  _Router,
+  Logger,
+}) {
+  describe('static methods', () => {
+    describe('factory', () => {
+      it('should return a Router instance', () => {
+        const config = utils.generateRouterConfig();
+        const Router = _Router.default(Paho.Paho);
+
+        const router = Router.factory(...config.asArray);
+
+        expect(router).not.toBeNil();
+        expect(router.constructor).not.toBeNil();
+        expect(router.constructor.name).toEqual('Router');
+      });
+      it('should throw with invalid arguments', () => {
```

Comment:
@brainthinks I think this test can be refactored/thrown out. I would just test the constructor to make sure the `bouncer` method is called. Then, thoroughly test bouncer method.